### ISSUE TITLE
OMeta parse errors get ignored by pybars3

### ISSUE THREAD

**Issue by isaacdd** [wbond/pybars3#23]
To reproduce:

```
>> from pybars import Compiler
>> compiler = Compiler()
>> text = "{{#test}}a{{/#test}}"
>> compiler.compile(unicode(text))
<function render at 0x275f2a8>  # no error reported here
```
After inspecting the parse error, it seems that OMeta only reports the position, but `None` for the error.
```
>> tree, error = compiler._handlebars(text).apply('template')
>>> error
ParseError(13, None)
```
I think a parse error should be raised whenever the parse position is less than the length of the string.

**Comment by dagnelies** [comment]
+1

**Comment by isaacdd** [comment]
@dagnelies This has been fixed in this fork: https://github.com/DataDog/pybars3.

**Comment by wbond** [comment]
@isaacdd Are you interested in having access to this repo and the project on PyPi? I'm too overcommitted with stuff right now to keep up with this.

**Comment by isaacdd** [comment]
@wbond Thanks! I would be interested in that.

**Comment by wbond** [comment]
@isaacdd Do you have a PyPi username?

**Comment by isaacdd** [comment]
@wbond it is `isadaqah`.

**Comment by wbond** [comment]
Ok, all set on GitHub and PyPi. Thanks for helping out!

**Comment by tomgross** [comment]
If you do a release can you include the other PR #19 #20 and #21 too, please?

**Comment by isaacdd** [comment]
Fixed with 0.9.2.

**Comment by dagnelies** [comment]
great, thanks @isaacdd !
